### DISCUSSION TITLE
Modularização

### DISCUSSION THREAD

**Discussion by CS-ConcreteAmorim** [concretesolutions/concrete_android_community#7] [Geral]
Como decidir qual é a melhor forma de Modularizar meu projeto?

**Comment by CS-ConcreteAmorim** [comment]
Olá! Seja Bem Vindo ao nosso Discussions! 

Consulte a nossa Wiki para compreender  qual é o melhor tipo de modularização para o seu projeto.

E não esquece de dar uma conferida no tópico de organização de Pacotes também, para ficar claro qual é opção é mais viável pra você!


<img width="1646" alt="Captura de Tela 2021-05-06 às 14 06 03" src="https://user-images.githubusercontent.com/80279421/117338010-75291c80-ae74-11eb-8f57-ebc5e11a8a5d.png">

**Comment by cs-cristianocortez** [comment]
Esse é um codelab falando sobre o assunto dos codelabs da Concrete: [Conceitos de modularização](http://concretesolutions.github.io/codelabs/conceitos-de-modularizacao/index.html?index=..%2F..index#6), com certeza vai te ajudar !!
